components/cosmetic_filters/resources/data/element_picker.html

```diff
@@ -1,165 +1,317 @@
+  }
 
-    .secondary-button {
-      background-color: #FFFFFF;
-      color: #4036d3;
-      cursor: pointer;
-      padding: 4px 6px;
-      flex-grow: 1;
-      border-width: 0px;      
-    }
-    
-    .secondary-button.dark {
-      color: #c1c4ff;
-    }
+  #rules-box>textarea {
+    box-sizing: border-box;
+    height: 100%;
+    width: 100%;
+    resize: none;
+    border: none;
+    padding: 5px;
+    overflow: auto;
+    overflow-wrap: break-word;
+    background-color: white;
+    color: var(--text-color);
+  }
 
-    .secondary-button.light {
-      color: #3E37D4
-    }
-    .secondary-button.disable.dark {
-      color: #c1c4ff;
-    }
+  #main-section.dark>#rules-box>textarea {
+    background-color: var(--dynamic-bg-color);
+  }
 
-    .secondary-button.disable.light {
-      color: #3E37D4
-    }
+  #main-section {
+    background-color: var(--theme-background-color);
+    border-radius: 16px;
+    color-scheme: light only;
+    bottom: 4px;
+    left: unset;
+    right: 4px;
+    width: min(360px, calc(100vw - 32px));
+    box-sizing: border-box;
+    padding-top: 56px;
+    padding-right: 16px;
+    padding-left: 16px;
+    position: fixed;
+    opacity: 1;
+    height: auto;
+  }
 
-    .secondary-button-bordered {
-      border-width: 1px;
-      border-color: #4036d3;
-    }
-  
-    svg {
-      cursor: crosshair;
-      box-sizing: border-box;
-      height: 100%;
-      width: 100%;
-      top: 0;
+  #main-section.desktop.minimized {
+    width: 56px;
+    padding: 0px;
+    height: 56px;
+    min-width: auto;
+    border-radius: 12px;
+    background-color: var(--primary) !important;
+  }
+
+  @media (orientation: portrait) {
+    #main-section:not(.desktop) {
+      bottom: 0;
+      right: 0;
       left: 0;
-      position: absolute;
-    }
-  
-    #darken {
-      fill: rgba(0, 0, 0, 0.5);
-    }
-  
-    #slider-container {
-      padding-bottom: 4px;
+      border-radius: 28px 28px 0 0;
+      padding-top: 41px;
+      border: none;
+      width: 100%;
     }
-    .target {
-      stroke: #3d39c8;
-      stroke-width: 2px;
-      fill: rgba(121, 116, 224, 0.25);
+  }
+
+  @media (orientation: landscape) {
+    #main-section:not(.desktop) {
+      padding-top: 36px;
     }
+  }
+
+  #main-section.minimized:not(.desktop) {
+    transition: transform 0.3s ease;
+    transform: translateY(calc(100% - 28px));
+  }
+
+  #main-section .drag-header {
+    position: absolute;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 28px;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    border-radius: 16px 16px 0 0;
+  }
+
+  #main-section.desktop .drag-header {
+    height: 48px;
+    justify-content: right;
+    background-color: var(--dynamic-bg-color);
+  }
+
+  #main-section:not(.desktop) .drag-indicator {
+    width: 36px;
+    height: 4px;
+    background-color: #AAAAAA;
+    border-radius: 2px;
+  }
+
+  #main-section.desktop .drag-indicator {
+    display: none;
+  }
```

Comment:
Let's try and avoid some of the noise:
```suggestion
  .drag-indicator {
    width: 36px;
    height: 4px;
    background-color: #AAAAAA;
    border-radius: 2px;
  }

  .desktop .drag-indicator {
    display: none;
  }
```